root/etc/e-smith/events/actions/nethserver-nextcloud-rh-mariadb105-dump-database

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
+#
+# Copyright (C) 2021 Nethesis S.r.l.
+# http://www.nethesis.it - nethserver@nethesis.it
+#
+# This script is part of NethServer.
+#
+# NethServer is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License,
+# or any later version.
+#
+# NethServer is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with NethServer.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+source /opt/rh/rh-mariadb105/enable
+PATH=$PATH:/opt/rh/rh-mariadb105/root/usr/bin/
```

Comment:
I'm not sure, we just need to verify it.
Shouldn't be the PATH already prepared from the source command?

Here we do not event export the path.

Reply:
done